src/components/Callout/InfoCallout.tsx

```diff
@@ -6,12 +6,14 @@ import { Callout } from './Callout'
 export const InfoCallout: React.FC<{
   className?: string
   noIcon?: boolean
+  icon?: JSX.Element
```

Comment:
added this to allow using the custom "owner" crown icon in the pay modal InfoCallout

Reply:
clean!